torch/fx/experimental/recording.py

```diff
@@ -277,7 +277,13 @@ def wrapper(*args, **kwargs):
                         raise
 
             except Exception:
-                log.error("failed while running %s(*%s, **%s)", name, args[1:], kwargs)
+                log.error(  # noqa: G201
+                    "failed while running %s(*%s, **%s)",
+                    name,
+                    args[1:],
+                    kwargs,
+                    exc_info=log.isEnabledFor(logging.INFO),
```

Comment:
@Skylion007 when exc_info is not unconditionally True, I think the lint saying you should `log.exception` is inappropriate, IMO